### ISSUE TITLE
Sounds and music are disabled

### ISSUE THREAD

**Issue by ScytheandGenesisGames** [OffBroadway/flippyboot-ipl#9]
Using the new boot.iso has the sounds disabled. if needed to adjust something on the GameCube's main menu no sound music plays - it might be disabled in the cube boot file

**Comment by trevor403** [comment]
Are you using GCLoader? Are there any other details you could provide about your setup?

**Comment by ScytheandGenesisGames** [comment]
> Are you using GCLoader? Are there any other details you could provide about your setup?

Yep if I let's say boot to Swiss-GC then needed to fix the clock no music or the sounds are played

I'm doing what it says to use the .dol version of the swiss-GC.dol which my console is NTSC
